src/emcc_wrapper.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+SCRIPT_FILE=$(readlink -f "$BASH_SOURCE")
+SCRIPT_DIR=$(cd $(dirname $SCRIPT_FILE) && pwd)
+
+export EMCC_SKIP_SANITY_CHECK=1
```

Comment:
It writes a file called <emconfig>_sanity alongside the emconfig (which is a non-writable location in our case).